EIPS/eip-3475.md

```diff
@@ -2,10 +2,9 @@
 eip: 3475
 title: Abstract Storage Bonds
 description: Interface for creating tokenized obligations with abstract on-chain metadata storage
-author: Yu Liu (@yuliu-debond), Varun Deshpande (@dr-chain), Cedric Ngakam (@drikssy), Dhruv Malik (@GrandGarcon), Samuel Gwlanold (@Edoumou), Toufic Batrice (@toufic0710)
+author: Yu Liu (@yuliu-debond), Varun Deshpande (@dr-chain), Cedric Ngakam (@drikssy), Dhruv Malik (@GrandGarcon), Samuel Gwlanold Edoumou (@Edoumou), Toufic Batrice (@toufic0710)
 discussions-to: https://ethereum-magicians.org/t/eip-3475-multiple-callable-bonds-standard/8691
-status: Last Call
-last-call-deadline: 2022-07-30
+status: Final
 type: Standards Track
 category: ERC
 created: 2021-04-05
```

Comment:
```suggestion
created: 2021-04-05
requires: 20, 721
```
```suggestion
created: 2021-04-05
requires: 20, 721, 1155
```

Depending on whether you specify compatibility with 1155 or not.

Reply:
ah its weird, in my current branch its already added status: Final